.github/workflows/e2e_creator_learner_dashboard_and_editor_tabs.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-build --skip-install --suite="learnerDashboard" --prod_env
         env: 
-          VIDEO_RECORDING_IS_ENABLED: 0
+          VIDEO_RECORDING_IS_ENABLED: 1
```

Comment:
Just leaving a blocking review to make sure we remember to turn videos off before merging

Reply:
Removed